### PR TITLE
chore : add missing artifacts to the bom, refresh JSON

### PR DESCRIPTION
**Pull request by cunningt**
Add missing camel-spring-boot artifacts to the camel-spring-boot BOM

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: tooling/camel-spring-boot-dependencies/pom.xml [hunk 1]
```diff
@@ -4170,142 +4170,142 @@
       <dependency>
         <groupId>org.apache.cxf.services.sts</groupId>
         <artifactId>cxf-services-sts-core</artifactId>
-        <version>3.5.2</version>
+        <version>3.5.1</version>
```
Comment (davsclaus) [inline tooling/camel-spring-boot-dependencies/pom.xml:4173]: You go backwards from 3.5.2 to 3.5.1 that is indicating this PR is oudated

File: tooling/camel-spring-boot-dependencies/pom.xml [hunk 2]
```diff
@@ -2652,6 +2652,11 @@
         <artifactId>camel-fastjson</artifactId>
         <version>3.17.0-SNAPSHOT</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>camel-fatjar-main</artifactId>
+        <version>3.17.0-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (cunningt) [reply tooling/camel-spring-boot-dependencies/pom.xml]: Refreshed Camel SNAPSHOT, rebuilt

File: tooling/camel-spring-boot-dependencies/pom.xml (hunk 1 above)
Comment (cunningt) [reply tooling/camel-spring-boot-dependencies/pom.xml]: Refreshed Camel SNAPSHOT, rebuilt

**Comment by cunningt** [comment]
Refreshed with an updated camel SNAPSHOT, and limited the commit to just the tooling changes.

**Review by davsclaus — APPROVED** [review]

**Pull request merged**